tests/functional/aws-node-sdk/test/object/compluteMpu.js

```diff
@@ -7,8 +7,8 @@ const bucket = `bucketcompletempu-bucket-${Date.now()}`;
 
 const parts = [];
 
-// results in body of 589824 bytes
-for (let i = 0; i < 100000; i ++) {
+// results in body of 1049793 bytes (1 MB = 1048576)
+for (let i = 0; i < 15600; i ++) {
```

Comment:
I think it makes sense to test a post size bigger than the limit but close to it.
Originally the number of parts being uploaded was 100,000 but this resulted in a post body size that was a lot larger than the limit, so even when we increased the limit the test was still passing.